src/VirtualClient/VirtualClient.Main/profiles/PERF-SQL-POSTGRESQL.json

```diff
@@ -65,6 +66,9 @@
             "Parameters": {
                 "Scenario": "StartAPIServer"
             }
+        },
+        {
+            "Type": "FormatDisks"
```

Comment:
I would put the format disk before the sql installation. I know right now we are putting it in the OS disk, but we will change the location later, so formatting disk first helps